### PR TITLE
Implement autosuggest sample

### PR DESCRIPTION
**Pull request by cvializ**
/to @sebastianbenz @bpaduch 
/cc @aghassemi 

I'm not a very strong Go developer so I welcome any feedback on the backend code : )

screen shot 2017-08-28 at 11 37 21 am
screen shot 2017-08-28 at 11 37 09 am

### CODE REVIEW

File: src/50_Samples_%26_Templates/Autosuggest.html [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    text-align: center;
+  }
+  </style>
+</head>
+<body>
+  <!-- ## Autosuggest form field -->
+  <!-- -->
+  <h2 class="p1">Autosuggest</h2>
+
+  <!--
```
Comment (kul3r4) [inline src/50_Samples_%26_Templates/Autosuggest.html]: Could you also mention how you combined amp-form, amp-list and amp-bind to implement the autosuggest?

Comment (cvializ) [reply src/50_Samples_%26_Templates/Autosuggest.html]: Ah yes that's a good idea

Comment (cvializ) [reply src/50_Samples_%26_Templates/Autosuggest.html]: Done!

**Pull request merged**